packages/proxy/src/logger.ts

```diff
@@ -0,0 +1,21 @@
+import * as winston from "winston";
+
+export const Logger = (function () {
+  let instance: winston.Logger;
```

Comment:
oh thank god. I was worried you would try to use `const`

Reply:
I actually debated about this for quite some time. `let` vs. `const` is quite a debate to be had! I ultimately decided to use `let` instead of `const` because I re-assign the variable, which is not possible when using `const`! Thus, `let` was the clear option here. However, now that I have thought about this more, we could just instantiate the "instance" from the start and use `const`. Or, we could keep it with `let` if we decide that we ever need to re-assign it. This is quite a conundrum and kept me up pretty late at night. In summary, it does not matter and we should consider using `var` as "instance" is a variable and not a `let` or `const`.